File: pinyougou_manager_web/src/main/webapp/js/controller/loginController.js
app.controller("loginController", function ($scope, $controller, loginService) {
    $controller("baseController", {$scope: $scope});

    //登录人用户展示
    $scope.login = function () {
        loginService.login().success(function (response) {
            $scope.username = response.username
        })
    }
});

File: pinyougou_merchant_web/src/main/java/com/pinyougou/merchant/controller/loginController.java
package com.pinyougou.merchant.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.pinyougou.sellergoods.service.SellerService;
import org.springframework.security.core.Authentication;
import org.springframework.security.core.context.SecurityContext;
import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.security.core.userdetails.User;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/login")
public class loginController {
    @Reference
    private SellerService sellerService;

    @RequestMapping("/user")
    public Map loginUser() {
        Map<String, String> map = new HashMap<>();
        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        map.put("username", name);
        return map;
    }
}
